kasa/aestransport.py

```diff
@@ -182,6 +182,9 @@ async def send_secure_passthrough(self, request: str) -> Dict[str, Any]:
             assert self._encryption_session is not None
 
         raw_response: str = resp_dict["result"]["response"]
+        if not raw_response:
```

Comment:
Was the {} response happening here in the transport before attempting decryption or subsequently in `smartprotocol`?  If here then there should probably be a similar check in `klaptransport` and probably also a check for `result` and `response` not being present as well.

Reply:
Alas, the device got updated so I'm not sure anymore about this. I think it's better to remove this one for now, and re-add later if needed.